caddyfile/dispenser.go

```diff
@@ -204,6 +204,13 @@ func (d *Dispenser) SyntaxErr(expected string) error {
 	return errors.New(msg)
 }
 
+// RewritePathErr returns an error indicating the path being rewritten to
+// is missing an initial /
+func (d *Dispenser) RewritePathErr() error {
```

Comment:
Since `rewrite` is simply a directive, I do not think it is best to have it's error as part of the dispenser. This looks out of place to me.